lib/uksched/include/uk/thread.h

```diff
@@ -106,6 +106,15 @@ struct uk_thread *uk_thread_current(void)
 	return *current;
 }
 
+static inline
+unsigned long uk_get_stack_bottom(void)
+{
+	unsigned long sp = ukarch_read_sp();
+
+	return sp & STACK_MASK_TOP;
+}
+
+
```

Comment:
Regarding changes in `uksched` we should wait for the sched/musl changes being upstream and then adapt as necessary.

Reply:
Yes I agree, do you have any better alternative for the owner, like a property of thread whose value is unique per thread ?